README.md

```diff
@@ -59,8 +59,8 @@ This will start a local development server on port 5000 (http://localhost:5000).
 
 ## Adding a project
 
-Codetribute can read from both GitHub and Bugzilla. To add a new entry to the site, create a file `<project-name>.yml` 
-in `src/data` using the [template example](#template-example) as the initial setup. 
+Codetribute can read from both GitHub and Bugzilla. Only bugzilla bugs that have `good-first-bug` keyword will appear in the list.
```

Comment:
Codetribute can read from both GitHub and Bugzilla. Regarding the latter, only bugs with the `good-first-bug` keyword will appear in the list.